modules/carto/src/sources/quadbin-query-source.ts

```diff
@@ -19,17 +26,19 @@ export const quadbinQuerySource = async function (
     aggregationExp,
     aggregationResLevel = 6,
     sqlQuery,
-    spatialDataColumn = 'quadbin:quadbin',
+    spatialDataColumn = 'quadbin',
     queryParameters
   } = options;
-  const urlParameters: UrlParameters = {aggregationExp, q: sqlQuery};
+  const urlParameters: UrlParameters = {
+    aggregationExp,
+    q: sqlQuery,
+    spatialDataColumn,
+    spatialDataType: 'quadbin'
```

Comment:
Use the type

Reply:
Actually we are moving towards just using the strings in the deck/loaders codebase. It makes it easier to read as you don't have to go and look at what the constant is and TypeScript ensures that it is valid anyway.

Reply:
ok